app/views/devise/mailer/_event_email.html.erb

```diff
@@ -0,0 +1,23 @@
+<p>Hi <%= user.first_name %>, you have a new email from <%= t 'global.organization_name' %>!</p>
+<% if !delete.blank? %>
+  <% header = I18n.t("global.deleted") %>
+<% else %>
+  <% update_state ? header = I18n.t("global.updated") : header = I18n.t("global.new") %>
+<% end %>
+<p><%= header %> <%= event.eventType %>:</p>
+
+<p><%= get_event_text_params(event, true).html_safe%></p>
+
+<% if delete.blank? %>
+
+  <% if Event.eventTypes[event.eventType].to_i < 2 %>
```

Comment:
There's a lot of magic numbers and it makes the code super confusing to read/understand. It would take me a while to edit it if I had to.